.travis.yml

```diff
@@ -14,5 +14,5 @@ before_install:
 
 script:
  - ./autogen.sh
- - ./configure --with-stage1-default-name="example.com/some-stage1-for-rkt" --with-stage1-default-version="0.0.1"
+ - ./configure
```

Comment:
Not sure if I like the change - it will be building three flavors now (coreos, kvm and fly). Maybe instead consider building `./configure --with-stage1-flavors=host,fly` - this should be all the go stuff we have, but will not download CoreOS PXE image nor download kernel and kvm and build them.